src/reanimated2/layoutReanimation/web/transition/Sequenced.web.ts

```diff
@@ -0,0 +1,42 @@
+import { parseAnimationObjectToKeyframe } from '../animationParser';
+import type { TransitionData } from '../animationParser';
+
+export function SequencedTransition(
+  name: string,
+  transitionData: TransitionData
+) {
+  const { translateX, translateY, scaleX, scaleY, reversed } = transitionData;
+
+  const scaleValue = reversed ? `1, ${scaleX}` : `${scaleY}, 1`;
+
+  // TODO: Change proportions
+  const sequencedTransition = {
+    name: name,
+    style: {
+      0: {
+        transform: [
+          {
+            translateX: `${translateX}px`,
+            translateY: `${translateY}px`,
+            scale: `${scaleX}, ${scaleY}`,
+          },
+        ],
+      },
+      50: {
+        transform: [
+          {
+            translateX: reversed ? `${translateX}px` : '0px',
```

Comment:
Do we need to specify the unit here (`0px`) or can we pass just `0`?

Reply:
We can pass just `0`. The way how it works is that parser will output `translateX(<arg>)`, so in that case it would be `translateX(0)` instead of `translateX(0px)`. I've kept `px` for consistency, but we can remove it if you'd like to.